src/com/codersbay/Main.java

```diff
@@ -4,6 +4,31 @@
 
     public static void main(String[] args) {
 
+        int[] unsorted = {6, 20, 8, 19, 56, 23, 87, 41, 49, 53};
+
+        unsorted = insertionSort(unsorted);
+
+        for (int i = 0; i < unsorted.length; i++) {
```

Comment:
Hier gibt's schon eine methode die uns schön den inhalt eines arrays in einen string verwandelt - die [Arrays.toString](https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/util/Arrays.html#toString(int%5B%5D)) Methode

Reply:
Oh sorry, da hab ich mich wohl verguckt : )